PersonApi/V1/Infrastructure/CorrelationConstant.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace PersonApi.V1.Infrastructure
+{
+    public static class CorrelationConstants
+    {
+        public const string CorrelationId = "x-correlation-id";
+    }
+
+}
```

Comment:
In the BaseApi template this file has been moved out of the "Infrastructure" folder into the root of the "V1" folder.